steps/tslint/0.0.3/step.yml

```diff
@@ -0,0 +1,50 @@
+title: Run tslint command
+summary: Runs `tslint` with the given args.
+description: Runs `tslint` with the given args.
+website: https://github.com/ibratoev/steps-tslint
+source_code_url: https://github.com/ibratoev/steps-tslint
+support_url: https://github.com/ibratoev/steps-tslint/issues
+published_at: 2017-03-29T12:29:13.942802666+03:00
+source:
+  git: https://github.com/ibratoev/steps-tslint.git
+  commit: 18387733461e9726473e7aa3cb5e209fbbb5a65f
+host_os_tags:
+- osx
+- ubuntu
+project_type_tags:
```

Comment:
Thanks for the heads up! It makes sense. Didn't know that I can skip `project_type_tags`.